ddtrace/contrib/internal/pymongo/client.py

```diff
@@ -45,47 +46,32 @@
 _DEFAULT_SERVICE = schematize_service_name("pymongo")
 
 
+# TODO(mabdinur): Remove TracedMongoClient when ddtrace.contrib.pymongo.client is removed from the public API.
```

Comment:
nit: Same question here about marking as deprecated

Reply:
We already have deprecation warnings for this. With @rachelyangdog's work this class will be removed from the public api in v3.0.0.

Reply:
Oh oops, you're right this is the internal patch.py